Exec/RegTests/TGReact/tgreact.inp

```diff
@@ -60,6 +60,6 @@ pelec.plot_massfrac = 1
 
 # PROBLEM PARAMETERS
 prob.u0 = 400.0
-prob.iname = "cantera/profiles.dat"
+prob.iname = "profiles.dat"
 prob.case_type = "reacting"
-prob.nx = 2048
+prob.nx = 256
```

Comment:
this switch to 256 was intentional?

Reply:
Yeah, just to reduce the size of the .dat file because it has to be committed for the test to run